src/Stryker.Core/Stryker.Core.UnitTest/Mutators/StatementMutatorTests.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Shouldly;
+using Stryker.Core.Mutators;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Mutators
+{
+    public class StatementMutatorTests
+    {
+        [Fact]
+        public void ShouldBeMutationlevelStandard()
+        {
+            var target = new StatementMutator();
+            target.MutationLevel.ShouldBe(MutationLevel.Standard);
+        }
+
+        public static IEnumerable<object[]> MutableStatements => new List<object[]>
+        {
+            new object[] { SyntaxFactory.BreakStatement() },
+            new object[] { SyntaxFactory.ContinueStatement() },
+            new object[] { SyntaxFactory.YieldStatement(SyntaxKind.YieldBreakStatement) },
+            new object[]
+            {
+                SyntaxFactory.YieldStatement(
+                    SyntaxKind.YieldReturnStatement,
+                    SyntaxFactory.LiteralExpression(SyntaxKind.NumericLiteralExpression, SyntaxFactory.Literal(1))
+                )
+            },
+            new object[] { SyntaxFactory.ReturnStatement() },
+            new object[] {
+                SyntaxFactory.IfStatement(
+                    SyntaxFactory.LiteralExpression(SyntaxKind.TrueLiteralExpression),
+                    SyntaxFactory.EmptyStatement()
+                )
+            },
+        };
+
+        [Theory]
+        [MemberData(nameof(MutableStatements))]
```

Comment:
We prefer to use InlineData instead of MemberData. And since the `member` is not used anywhere else I'd like to see InlineData here as well

Reply:
I used MemberData because the statements aren't constants, and I cannot construct them inside InlineData (unless I'm missing something)

Reply:
If you look at other tests you'll see that we place the syntax as string in InlineData and parse it to syntax in the unit test. This is more readable because you can clearly see what syntax we're testing instead of having to build up a mental model from a SyntaxTree.

Reply:
Ok, changed